### DISCUSSION TITLE
Mobile context menu

### DISCUSSION THREAD

**Discussion by laawn** [jamiebrynes7/obsidian-todoist-plugin#127] [Q&A]
So now there's mobile support, how do we open the context menu in the mobile apps to open the task in the Todoist app?

**Comment by laawn** [comment]
@jamiebrynes7 I can now see the context menu on mobile but "Open task in Todoist" doesn't do anything on iOS. However, I can complete tasks via the context menu.

**Reply by laawn** [reply]
@jamiebrynes7 is this on the roadmap?

**Reply by jamiebrynes7** [reply]
Raised an issue here - https://github.com/jamiebrynes7/obsidian-todoist-plugin/issues/158
